### PR TITLE
fix hook order race with telemetry

### PR DESCRIPTION
**Pull request by trilopin**
#### Summary
Fixes hook race condition for telemetry (onactivate, onconfigchanged)
I've manually tested it against jira cloud.

mattermost-server and pluginapi deps have been updated, opengraph forced as an indirect dep to avoid a ambiguous dep conflict

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-51548

### CODE REVIEW

File: webapp/package-lock.json [hunk 1]
```diff
@@ -2625,7 +2625,7 @@
     "amdefine": {
       "version": "1.0.1",
       "resolved": "https://registry.npmjs.org/amdefine/-/amdefine-1.0.1.tgz",
-      "integrity": "sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=",
```
Comment (mickmister) [inline webapp/package-lock.json:2628]: I wonder how all these changes were introduced

File: server/telemetry.go [hunk 2]
```diff
@@ -63,3 +68,29 @@ func (p *Plugin) OnSendDailyTelemetry() {
 
 	p.TrackEvent("stats", args)
 }
+
+// Initialize telemetry setups the tracker/clients needed to send telemetry data.
+// The telemetry.NewTrackerConfig(...) param will take care of extract/parse the config to set rge right settings.
```
Comment (mickmister) [inline server/telemetry.go:73]:
```suggestion
// The telemetry.NewTrackerConfig(...) param will take care of extract/parse the config to set the right settings.
```

**Review by mickmister — APPROVED** [review]
LGTM :+1:

File: webapp/package-lock.json (hunk 1 above)
Comment (trilopin) [reply webapp/package-lock.json:2628]: It's been automatically added after install and local deploy. I can remove them from the PR but I'm afraid they'll come again and again in future PRs.

**Review by m1lt0n — APPROVED** [review]

**Comment by trilopin** [comment]
@DHaussermann Even if I'm not asking QA review for telemetry PRs, I'd want you to taste this one since I found (and fixed) some panics doing exploratory testing.

**Comment by m1lt0n** [comment]
@trilopin this doesn't have a milestone attached to it. The next milestone is 3.3.0, but there are still several pending items for that. Should a dot release be scheduled for it? cc: @mickmister @hanzei @asatkinson

**Pull request merged**